### DISCUSSION TITLE
pi-Base community call Fri 2025 Jan 24 at 4pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#1076] [Announcements]
Agenda

- pi-Base tutorial next month https://github.com/orgs/pi-base/discussions/1077
- Clean up "Squash and Merge" commit messages?
- Recommendation to avoid cryptic PR titles?
- Experimenting with ID "badges": https://stevenclontz-ids-when-typese.topology.pages.dev/dev/preview and https://github.com/pi-base/web/pull/194
- [Discussions vs Issues](https://github.com/pi-base/data/issues/1071#issuecomment-2539876981)
- Meta-properties https://github.com/pi-base/data/issues/1071
- importance (or not) of language/grammar
- Approving but not merging?

Zoom link: https://southalabama.zoom.us/j/93833246214

Previous meeting: https://github.com/orgs/pi-base/discussions/846

Next meeting: https://github.com/orgs/pi-base/discussions/1210

**Comment by prabau** [comment]
@StevenClontz I don't want to attract attention by creating usable links to it, but you may want to check the spam comment added at the end of discussion number 654 (He also created a bogus issue).  If possible, block that account.

I am sure this is against some rules of conduct of github, and he could maybe be banned sitewide.

**Reply by StevenClontz** [reply]
Thanks - I saw the spam message and reported it as such. It's been hidden but not yet deleted (in case that makes it more difficult for moderators to verify the spam).

**Reply by prabau** [reply]
Hmm, seems like pi-base is being targeted.  You may have seen another spam comment from another account which just joined github three hours ago. :-(

**Comment by prabau** [comment]
@StevenClontz Can you give @pzjp write access to the pi-base repository?  He has made great contributions so far.

**Reply by StevenClontz** [reply]
Done! Thanks @pzjp for your contributions. Please start creating your branches directly on the repository so it triggers automatic builds for your pull requests.

**Reply by pzjp** [reply]
Whenever this new permissions actually start working, should I abandon my fork and create branches directly in _pi-base/data_ or it's recommended to do this copying trick from the tutorial video?

**Reply by prabau** [reply]
@pzjp For #1156 that is currently in progress, we can continue with your repository for now.
For your new #1164, it would be preferable to create a new branch in the official repository, for example by using the "copying trick".  Then continue in that new branch.  You will have to close your PR and open a new PR based on the cloned branch, because github does not allow to change the branch within the same PR.

And for later PRs, they can be done directly from branches in the official repository.  So you won't be using your private pi-base repository anymore.

**Reply by StevenClontz** [reply]
Also I forgot to mention: your permissions are instant, but you need to accept the invitation (should be in your email and your GitHub notifications).

**Reply by pzjp** [reply]
This notification was surprisingly hidden...
Thank you!

**Comment by prabau** [comment]
@StevenClontz The last two builds in https://github.com/pi-base/data/actions have failed in the "upload bundle to R2" stage.  Seems an AWS problem.  Any idea?
